lib/updatehub/device.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DEVICE_H__
```

Comment:
Change to __UPDATEHUB_DEVICE_H__ as __DEVICE_H__ might conflict.

Reply:
It actually seems that include protector in this case should be named something like this `ZEPHYR_INCLUDE_UPDATEHUB_DEVICE_H_`, see existing header files in include dir for examples.